### ISSUE TITLE
Email::Simple can not send decoded strings

### ISSUE THREAD

**Issue by KES777** [rjbs/Email-Simple#22]
Please add into documentation that Email::Simple sends bytes. And if developer want to send `utf8` then he must `$body =  Encode::encode( utf8 => $body )`. 
```
 Devel::Peek::Dump( $body )
SV = PVIV(0x10c77a18) at 0xc3ff930
  REFCNT = 2
  FLAGS = (POK,IsCOW,pPOK)
  IV = 0
  PV = 0x10df0750 "## EXCEPTION MESSAGE\nRU Error message: \320\232\320\260\320\272\320\276\320\271-\321\202\320\276 \321\202\320\265\320\272\321\201\321\202 at /home/kes/work/projects/tucha/monkeyman/lib/MaitreD.pm line 75.\n\n"\0
  CUR = 134
  LEN = 136
  COW_REFCNT = 0
```

Otherwise some servers may not send email and respond with **timeout** if body with `UTF8` flag:
```
Devel::Peek::Dump( $body )
SV = IV(0xc3ff920) at 0xc3ff930
  REFCNT = 2
  FLAGS = (ROK)
  RV = 0x10d0d530
  SV = PVMG(0x10d36f60) at 0x10d0d530
    REFCNT = 5
    FLAGS = (OBJECT,POK,IsCOW,pPOK,UTF8)
    IV = 0
    NV = 0
    PV = 0x10d69cd0 "## EXCEPTION MESSAGE\nRU Error message: \320\232\320\260\320\272\320\276\320\271-\321\202\320\276 \321\202\320\265\320\272\321\201\321\202 at /home/kes/work/projects/tucha/monkeyman/lib/MaitreD.pm line 75.\n\n"\0 [UTF8 "## EXCEPTION MESSAGE\nRU Error message: \x{41a}\x{430}\x{43a}\x{43e}\x{439}-\x{442}\x{43e} \x{442}\x{435}\x{43a}\x{441}\x{442} at /home/kes/work/projects/tucha/monkeyman/lib/MaitreD.pm line 75.\n\n"]
    CUR = 134
    LEN = 136
    COW_REFCNT = 1
    STASH = 0x35583d0	"Mojo::ByteStream"
```

More dumps here: 
https://stackoverflow.com/a/60428013/4632019

**INFO**  
My postfix server was configured as:
```
strict_7bit_headers = no
strict_8bitmime = no
strict_8bitmime_body = no
```

**Comment by pali** [comment]
Hi @KES777! You are right _that Email::Simple sends bytes_. It is is not mentioned in documentation explicitly (which can be improved, I agree), but can be implied from the description of the module _simple parsing of RFC2822 message_. RFC2822 does not support 8bit, only 7bit ASCII.

> And if developer want to send utf8 then he must $body = Encode::encode( utf8 => $body ).

This is incorrect. You even cannot use UTF-8 as is in email messages. You need to encode UTF-8 then to 7bit, e.g. via quoted printable or base64.

Some SMTP servers supports 8bit, so in this case UTF-8 can be passed directly. But whole email delivery would work only in case *all* SMTP server on the route must support it -- having it only on your first hop SMTP server is not enough.

To overcome all these problems, there is https://metacpan.org/pod/Email::MIME module which takes Unicode strings (as developers would like to use, no Encode::encode is needed) and it formats email correctly, so can be passed to any SMTP server. Email::MIME is just extension to the Email::Simple which do all needed Encode::encode, base64, quote printable, etc.. encoding. So it is something which you want to use instead of Email::Simple. Email::Simple is there just for implementing other modules like Email::MIME (or for those who already read whole RFC2822 and want to do all encoding stuff manually) or Email::Stuffer

**Comment by jidanni** [comment]
Yes the man page should warn that unless one is using pure ASCII, this module will create illegal mail.
